libs/application/templates/party-letter/src/forms/messages.ts

```diff
@@ -0,0 +1,24 @@
+import { defineMessages } from 'react-intl'
+
+export const m = defineMessages({
+  companySection: {
+    id: 'pl.application:company.section',
```

Comment:
oooopss, ParentalLeave is already using `pl.application`. I belive if you run `extract-strings`, you will deprecate all the parentalLeave strings inside the existing contentful entry.

What about `ple.application` or `pal.application`, just something different 😅

Reply:
ple is fine, missed that